code/index.html

```diff
@@ -1,22 +1,82 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>A birds eye view</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Concert+One&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+
+  <header class="header-container">
+    <img class="header-logo" src="./logo.svg">
+    <nav class="navigation">
+      <a class="nav-link" href="">Access</a>
+      <a class="nav-link" href="">Anlita oss</a>
+      <a class="nav-link" href="">Annonsera</a>
+      <a class="nav-link" href="">Sök</a>
+    </nav>
+  </header>
+
+
+  <section class="big-news-section">
+    <h3>Big News</h3>
+    <div class="polaroid-big">
+      <img
+        src="https://images.unsplash.com/photo-1514412076816-d228b5c0973c?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80"
+        alt="Flock of birds over water" style="width:100%">
```

Comment:
move styling to css file. avoid inline styling (in the html file) as much as possible.